scripts/install.sh

```diff
@@ -325,16 +325,17 @@ if [[ $enable_self_signed_ca == "true" ]]; then
   # comment out vars
   configMapGeneratorStartLine=$(grep -n configMapGenerator ./default/kustomization.yaml |cut -d':' -f1)
   configMapGeneratorBeforeLine=$((configMapGeneratorStartLine-1))
-  sed "1,${configMapGeneratorBeforeLine}s/^/#/g" -i default/kustomization.yaml
-  
+  sed -i.bak "1,${configMapGeneratorBeforeLine}s/^/#/g" default/kustomization.yaml
+
   # remove webhookcainjection_patch.yaml
-  sed 's+- webhookcainjection_patch.yaml++g' -i default/kustomization.yaml
+  sed -i.bak  's+- webhookcainjection_patch.yaml++g' default/kustomization.yaml
 
   # create dummy secret 'modelmesh-webhook-server-cert'
   secretExist=$(kubectl get secret modelmesh-webhook-server-cert --ignore-not-found|wc -l)
-  if [[ $secretExist == 0 ]]; then
+  if [[ $secretExist -eq 0 ]]; then
     kubectl create secret generic modelmesh-webhook-server-cert 
   fi
+  rm default/kustomization.yaml.bak
```

Comment:
why don't we add `*.bak` into .gitignore file?

Reply:
we could add `*.bak` to `.gitignore` (and `.dockerignore`) but we should not accumulate temp files in the project directory